language.md

```diff
@@ -497,6 +497,19 @@ Stop the script after _integer_ seconds. The usual integer suffixes, especially
 `minutes` and `hours` can be added here. When a script runs over its time
 budget, the Prometheus variable `shesmu_run_overtime` will be set. 
 
+### Frequency
+
+- `Frequency` _integer_ `;`
+
+Run the script every _integer_ seconds. The usual integer suffixes, especially `minutes`
+and `hours` can be added here. 
```

Comment:
This is a bit unclear - maybe indicate that it's `seconds` by default, but `minutes` and `hours` could be explicitly specified?